msteams-platform/toolkit/add-How-to-guides-v5.md

```diff
@@ -16,7 +16,7 @@ Microsoft Teams Toolkit project templates focus on getting started with an app t
 
 |**Capability** | **How-to Guide** |
 |----------|----------|
-| Configure Tab Capability | [How to configure Tab capability within your Teams app](https://github.com/OfficeDev/TeamsFx/wiki/How-to-configure-Tab-capability-within-your-Teams-app) |
+| Configure Tab Capability | [How to configure Tab capability within your Teams app](configure-tab-capability.md) |
```

Comment:
The original document is out of date.
1. The Tab app created by Teams Toolkit now is hosted by Azure Static Web App, instead of Azure Storage Account.
2. The configurable tab is not necessary any more, since static tab can also be installed to a team or a group chat.

Reply:
The original document is updated.

Reply:
Updated.